bin/prettify-manifests.js

```diff
@@ -12,6 +13,6 @@ const manifestFiles = glob.sync(`packages/*/oclif.manifest.json`)
 for (const file of manifestFiles) {
   console.log(`Prettifying ${file}...`)
   const content = fs.readFileSync(file)
-  const prettyContent = JSON.stringify(JSON.parse(content),null,2).replaceAll(root, '.')
+  const prettyContent = stringify(JSON.parse(content), {space: '  '}).replaceAll(root, '.')
```

Comment:
why do we need a deterministic json stringifier here?

Reply:
Naive JSON stringification is non-deterministic. Hence, we ended up having builds fail because the commands were being reordered each time. This keeps the builds from failing.

Honestly I'm not totally sure why the oclif upgrade triggered this issue, maybe just a matter of a bigger JSON object to stringify causing race conditions?

As an aside, note the flags are now alphabetized, which is possibly a nice thing.